test/unit/san.c

```diff
@@ -72,20 +79,15 @@ TEST_BEGIN(test_guarded_large) {
 	}
 
 	/* Verify the pages are not continuous, i.e. separated by guards. */
-	uintptr_t min_diff = (uintptr_t)-1;
```

Comment:
I figured this is unnecessary, since if all diffs are less than expected value, then min must be less too.